src/components/QRCode/QRCodePassword/QRCodePassword.vue

```diff
@@ -22,6 +22,10 @@
                                         <input
                                             v-model="formItems.password"
                                             v-focus
+                                            onselectstart="return false"
```

Comment:
Should we create a new component `PasswordInput` and have these all common properties in the new component with defaults? And use that 'PasswordInput' component everywhere?